__tests__/components/editor/PropertyLabel.test.js

```diff
@@ -0,0 +1,89 @@
+// Copyright 2019 Stanford University see Apache2.txt for license
+
+import 'jsdom-global/register'
+import React from 'react'
+import { shallow } from 'enzyme'
+import PropertyLabel from '../../../src/components/editor/PropertyLabel'
+import { OverlayTrigger } from 'react-bootstrap'
+import RequiredSuperscript from '../../../src/components/editor/RequiredSuperscript'
+
+
+describe('<PropertyLabel />', () => {
+
```

Comment:
I find these test descriptions helpful and descriptive - I can understand what is supposed to be happening a lot better.